src/pages/Models/Details/ModelDetails.js

```diff
@@ -269,51 +200,44 @@ const ModelDetails = () => {
   const handleAppRowClick = (e, app) => {
     const currentApp = cloneDeep(app);
     currentApp.name = e.currentTarget.dataset.app;
-    setSlidePanelData({ currentApp });
     setFilterByApp(currentApp.name);
+    setPanelType("apps");
+    setEntity(currentApp.name);
+  };
+
+  const closePanel = () => {
+    history.push(`/models/${modelName}`);
+    setPanelType(null);
+    setEntity(null);
   };
 
   const applicationTableRows = useMemo(
     () =>
       generateApplicationRows(modelStatusData, handleAppRowClick, baseAppURL),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [baseAppURL, modelStatusData]
   );
 
   const unitTableRows = useMemo(
     () => generateUnitRows(modelStatusData, baseAppURL),
     [baseAppURL, modelStatusData]
   );
-  const unitSlidePanelRows = useMemo(
-    () => generateUnitRows(filteredModelStatusData, baseAppURL),
-    [baseAppURL, filteredModelStatusData]
-  );
 
   const machinesTableRows = useMemo(
     () => generateMachineRows(modelStatusData),
     [modelStatusData]
   );
-  const machinesSlidePanelRows = useMemo(
-    () => generateMachineRows(filteredModelStatusData),
-    [filteredModelStatusData]
-  );
 
   const relationTableRows = useMemo(
     () => generateRelationRows(modelStatusData, baseAppURL),
     [modelStatusData, baseAppURL]
   );
-  const relationSlidePanelRows = useMemo(
-    () => generateRelationRows(filteredModelStatusData, baseAppURL),
-    [filteredModelStatusData, baseAppURL]
-  );
-
-  const appSlidePanelHeader = useMemo(
-    () => generateAppSlidePanelHeader(slidePanelData.currentApp, baseAppURL),
-    [slidePanelData.currentApp, baseAppURL]
-  );
 
   const slidePanelActive = Object.entries(slidePanelData).length > 0;
```

Comment:
This is being used for the aria disabled toggle but I don't see how it can result in anything but being always 'enabled' as no value is ever defined for `slidePanelData`.